[RUM-12388] [V3] Support SVG in Session Replay (#1026)
DataDog/dd-sdk-reactnative: packages/react-native-session-replay/README.md

```js
module.exports = {
  presets: ['module:@react-native/babel-preset'],
  plugins: [
    [
      '@datadog/mobile-react-native-babel-plugin',
      {
        sessionReplay: {
          // SVG tracking is disabled by default
          // Set to true to enable SVG asset extraction
          svgTracking: true
        }
      }
    ]
  ]
};
```

### Setup Metro Plugin

Configure the Metro plugin in your `metro.config.js` to enable automatic SVG asset bundling:

```js
const { withSessionReplayAssetBundler } = require('@datadog/mobile-react-native-session-replay/metro');

module.exports = withSessionReplayAssetBundler({
  /* your existing Metro config */
});
```

The Metro plugin automatically monitors and bundles SVG assets during development and production builds.

### Installation Workflow

When setting up your project or after installing new dependencies, follow this workflow to ensure SVG assets are properly generated for native builds:

```sh
# 1. Install dependencies
yarn install

# 2. Generate Session Replay SVG assets
npx datadog-generate-sr-assets

# 3. Install iOS pods (if building for iOS)
cd ios && pod install && cd ..

# 4. Run your app
yarn ios
# or
yarn android
```

> The `datadog-generate-sr-assets` CLI utility scans your codebase for SVG elements and pre-generates optimized assets that will be included in your native builds.

Reviewer: ```suggestion
The `datadog-generate-sr-assets` CLI utility scans your codebase for SVG elements and pre-generates optimized assets that are included in your native builds.
```
